lib/krikri/entity_behaviors/aggregation_entity_behavior.rb

```diff
@@ -0,0 +1,12 @@
+
+module Krikri
+  class AggregationEntityBehavior < Krikri::EntityBehavior
+    def generated_entities
+      @activity.generated_entity_uris.lazy.map do |uri|
+        agg = DPLA::MAP::Aggregation.new(uri)
+        agg.get   # slow?
```

Comment:
Thanks for that idea.  I'm pushing a commit that will add this `load` argument to both methods.